src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/tests/fields/SmartContent.test.js

```diff
@@ -82,7 +82,7 @@ test('Should correctly initialize SmartContentStore', () => {
         sortMethod: undefined,
         tagOperator: undefined,
         tags: undefined,
-        types: undefined,
+        types: ['default', 'homepage'],
```

Comment:
When I said that, I thought that should also be asserted afterwards somehow :see_no_evil: Don't we at least check if the props of the child components are passed correctly in this test?

Reply:
No the only check we do in this test is 
`expect(SmartContentStore).toBeCalledWith('media', value, undefined, 'collections', undefined, schemaOptions, 'sulu_io');`
but we don't explicitly check the props

Reply:
I'm not sure but I don't see any other possibility to test this. The `SmartContent` field passes the `types` via the `value` object to the `SmartContentStore` and this is already checked with the line from my comment above. The field-type doesn't have any other reference to the `types`.